.github/workflows/test-fast.yml

```diff
@@ -95,6 +95,18 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ inputs.go-version }}
+      - id: go-cache-paths
+        run: |
+          echo "::set-output name=go-build::$(go env GOCACHE)"
+          echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+      - name: Go Cache
+        uses: actions/cache@v2
+        id: go-cache
+        with:
+          path: |
+              ${{ steps.go-cache-paths.outputs.go-build }}
+              ${{ steps.go-cache-paths.outputs.go-mod }}
+          key: ${{ matrix.platform }}-go-cache-${{ hashFiles('**/go.sum') }}
```

Comment:
Possibly may need `test-subset` in the cache key.

Reply:
Do you think we should just refactor matrix.platform to matrix.os here and add -latest everywhere it's used so we can ensure cache reuse?

Reply:
It's possible.. But I'm also wondering if distinct keys are actually a good thing, since workloads exercise different subset of the builds, the contents of the cache are slightly different, and if these different workloads compete for what's written into the shared cache key maybe it's better to keep them separate. At the cost of having more data in the cache. I do not know what the limits/costs are of having more data in the cache yet, but perhaps that is not yet a factor at all?

Reply:
I don't yet know enough about GitHub's cache to weigh in either, good things for us to consider going forward.